azure-auth-helper/src/test/java/com/microsoft/azure/auth/configuration/AuthConfigurationTest.java

```diff
@@ -0,0 +1,125 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.auth.configuration;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class AuthConfigurationTest {
+    @Test
+    public void testValidate() {
+        AuthConfiguration auth = new AuthConfiguration();
```

Comment:
Add some comments here like what error will be throw here , as error count could be a bit confusing.